packages/web/src/utils/commonPasswordCheck.ts

```diff
@@ -46,3 +46,7 @@ export const commonPasswordCheck = async (
 
   return commonPasswordList.test(password) as boolean
 }
+
+export const isNotCommonPassword = async (password: string) => {
+  return !(await commonPasswordCheck(password))
+}
```

Comment:
😂

Reply:
(I do think this is actually fine, just funny)

Reply:
i desparately wanted to pass a single function to zod, lol